src/max_number_occurence.py

```diff
@@ -33,6 +35,8 @@ def max_number_occurence(generated_dictionary):
         .format(number = result[0], repeats = result[1])
```

Comment:
extra spaces arount `=` are not needed - https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements search for `Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value. `